airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/JobCreationAndStatusUpdateActivityImpl.java

```diff
@@ -88,6 +95,17 @@ public JobCreationOutput createNewJob(final JobCreationInput input) {
     }
   }
 
+  private void emitSrcIdDstIdToReleaseStagesMetric(final UUID srcId, final UUID dstId) throws IOException {
+    final var releaseStages = configRepository.getDatabase().query(ctx -> MetricQueries.srcIdAndDestIdToReleaseStages(ctx, srcId, dstId));
+    if (releaseStages == null) {
+      return;
+    }
+
+    for (final ReleaseStage stage : releaseStages) {
+      DogStatsDMetricSingleton.count(AirbyteMetricsRegistry.JOB_CREATED_BY_RELEASE_STAGE, 1, MetricTags.RELEASE_STAGE + stage.getLiteral());
```

Comment:
Instead of leaving it up to the developer to add the `MetricTags.RELEASE_STAGE` and `stage.getLiteral()` strings together, what if we defined a helper in the `MetricsTags` class that did this for us? 

`MetricTags.tagForReleaseStage(stage)` or something like that. I think I like this as a convention because it avoids accidentally using two different tags in two different places, like if one developer accidentally adds a space in between `release_stage:` and the stage literal.

Reply:
I like that idea.